conformance_tests/sysman/test_sysman_init/src/test_init_sysman_from_sysman_handle.cpp

```diff
@@ -0,0 +1,43 @@
+/*
+ *
+ * Copyright (C) 2024 Intel Corporation
+ *
+ * SPDX-License-Identifier: MIT
+ *
+ */
+
+#include "gtest/gtest.h"
+
+#include "logging/logging.hpp"
+#include "utils/utils.hpp"
+#include "test_harness/test_harness.hpp"
+
+#include <level_zero/zes_api.h>
+
+namespace {
+
+TEST(SysmanInitTests,
+     GivenCoreAndSysmanInitializedThenSysmanHandleWorksForEnumFrequencyDomanis) {
```

Comment:
GivenCoreAndSysmanInitializedWhenRetrievingFrequencyDomainsWithSysmanDeviceHandleThenSuccessIsReturned ?

Reply:
Changing to 
``` GivenSysmanInitializedFromZeInitAndZesInitThenSysmanHandleFromZesInitReturnsSuccessForEnumFrequencyDomains ```
Please check this as well.